frontend/packages/shared/src/mocks/queriesMock.ts

```diff
@@ -159,6 +159,7 @@ export const queriesMock: ServicesContextProps = {
 
   // Queries - PrgetBpmnFile
   getBpmnFile: jest.fn().mockImplementation(() => Promise.resolve<string>('')),
+  getProcessTaskType: jest.fn().mockImplementation(() => Promise.resolve<string>('')),
```

Comment:
Nice, I loved that you added a "negative" value (empty string in this case). This forces us to mock `getProcessTaskType` with the desired result in each test. 🚀 👏